### DISCUSSION TITLE
Include the expires field in the "Creating Order" endpoint

### DISCUSSION THREAD

**Discussion by Lurkars** [pretix/pretix#2986] [Ideas] [answered]
As title suggest, it would be nice to include the expires field to set expiry of the order during creation. In current state this leads to problem, that if an order should be created via API and send_mail feature with a different expiry than default, the expire date in the email will always be wrong, because email gets send before change from extends endpoints applies.

**Comment by Lurkars** [comment]
I understand that this has no high priority, but just wanted to send a short ping here that this is still an issue for me.

**Reply by raphaelm** [reply, marked as answer]
Apparently I hoped someone would just create a PR for this when I saw it last year :stuck_out_tongue: Here it is: 7f0ed374b
